assets/airports/eidw.json

```diff
@@ -875,136 +1072,578 @@
+            "draw": [
+                ["DW554", "DW555", "DEXEN"],
+                ["OE", "_DEXEN1", "_DEXEN2", "_DEXEN3", "_DEXEN4", "_DEXEN5", "_DEXEN6", "_DEXEN7", "_DEXEN8", "_DEXEN9", "_DEXEN10", "_DEXEN11", "_DEXEN12", "_DEXEN13", "_DEXEN14", "_DEXEN15", "_DEXEN16", "_DEXEN17", "_DEXEN18", "_DEXEN19", "_DEXEN20", "_DEXEN21", "_DEXEN22", "_DEXEN23", "_DEXEN24", "_DEXEN25", "_DEXEN26", "_DEXEN27", "_DEXEN28", "_DEXEN29", "_DEXEN292", "DEXEN"],
+                ["DWS81", "DEXEN"]
+            ]
         },
         "INKUR": {
             "icao": "INKUR",
             "name": "Inkur",
-            "suffix": {"10":"6E", "28":"5A", "16":"5M", "34":"1G"},
+            "suffix": {"10":"6E", "16":"5M", "28":"5A", "34":"1G"},
             "rwy": {
-                "10": [["DW551", "A30+"], "LEMTA", "DODIG"],
-                "28": [["OE", "A30++"]],
-                "16": [["DWS77", "A30+"], "DWS78", "NITIL"],
-                "34": [["DWN26", "A30+"], "DWW06"]
+                "10": [["^DW551", "A30+"], "LEMTA", "DODIG"],
+                "16": [["^DWS77", "A30+"], "DWS78", "NITIL"],
+                "28": [["^OE", "A30+"]],
+                "34": [["^DWN26", "A30+"], "DWW06"]
+            },
+            "body": [],
+            "exitPoints": {
+                "INKUR": ["INKUR"]
             },
-            "body": ["INKUR"],
-            "draw": [["LEMTA", "DODIG", "INKUR"], ["OE", "INKUR"], ["DWS77", "DWS78", "NITIL", "INKUR"], ["DWN26", "DWW06", "INKUR"]]
+            "draw": [
+                ["LEMTA", "DODIG", "INKUR"],
+                ["_LIFFY5", "INKUR"],
+                ["DWS77", "DWS78", "NITIL"],
+                ["DWN28", "INKUR"],
+                ["_BAL11", "DWW06", "INKUR"]
+            ]
         },
-        "DEXEN": {
-            "icao": "DEXEN",
-            "name": "Dexen",
-            "suffix": {"10":"2E", "28":"1A", "16":"1M"},
+        "INKUR(AB)": {
+            "icao": "INKUR(AB)",
+            "name": "Inkur",
+            "suffix": {"10":"6F", "16":"5P", "28":"5B"},
             "rwy": {
-                "10": [["DW554", "A30+"], "DW555"],
-                "28": [["OE", "A30+"]],
-                "16": [["DWS77", "A30+"], "DWS81"]
+                "10": ["DW931", "LEMTA", "DODIG", "INKUR"],
+                "16": ["OE"],
+                "28": ["DUB", "DWN28"]
+            },
+            "body": [],
+            "exitPoints": {
+                "INKUR": ["INKUR"]
             },
-            "body": [["DEXEN", "A120+"]],
-            "draw": [["DW554", "DW555", "DEXEN"], ["OE", "DEXEN"], ["DWS81", "DEXEN"]]
+            "draw": [
+                ["INKUR"]
+            ]
+        },
+        "KISHA": {
+            "icao": "KISHA",
+            "name": "Keeshah",
+            "suffix": {"10":"2F", "16":"1P", "28":"1B", "34":"1H"},
+            "rwy": {
+                "10": ["DW931"],
+                "16": ["OP", "BATED"],
+                "28": ["DUB"],
```

Comment:
As far as I know, the AIP won't update the chart unless a change has been made. This explains why the date shows 2012, but the chart is still valid for the current airac cycle. In any case, DW931 on the DUB 096 radial so aircraft will fly over it even if it's not on their FMC. Same goes for ERUDA.